.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,21 @@
+i---
+name: Feature request
+about: Suggest an idea for this project
+
+---
+
+<!-- NOTE: ⚠️ For larger proposals, we follow the CAEP process as outlined in https://sigs.k8s.io/cluster-api/CONTRIBUTING.md. -->
```

Comment:
this makes me wonder if we should add a `docs/enhancements` directory to the repo?

or maybe just make a reference that we would like them to go there (if and when we have one)

Reply:
good point, added just in case